Assignments/Dockerfile

```diff
@@ -0,0 +1,29 @@
+
+FROM python:3.8.2
+
+WORKDIR /app
+#replace with a volume
+#COPY baseball.sql .
+
+
+
+
+RUN apt-get update && apt-get -y install default-mysql-client \
```

Comment:
I'm jealous you got this to work. I had a bunch of trouble and had to go with a different base image container